### PR TITLE
doc: OVHcloud account SSO connection

### PR DESCRIPTION
**Pull request by sebferrer**
| Question         | Answer
| ---------------- | ---
| Branch?          |  doc/user-account-connect-saml-sso
| Bug fix?         |  no
| New feature?     |  yes
| Breaking change? |  no
| Tickets          |  ECIAM-1
| License          |  BSD 3-Clause

## Description

OVHcloud Account as service provider in federation.
This feature allows the user to link his OVH account to an identity provider in order to perform an SSO authentication.
More information in the Jira issue.

### :flags: Feature

- https://github.com/ovh/manager/pull/7527

### :flags: Translations

- [ ] TRANS-46547

### CODE REVIEW

**Comment by OVH-Team-Guides** [comment]
@sebferrer , FR translation now added. Please review it.
Small addiitons were also added onto the en-us version and duplication has been done.
Account "product" pages have also been prepared with the new "Advanced use" section.

File: pages/account/customer/ovhcloud-account-connect-saml-sso/guide.fr-fr.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+
+## Objectif
+
+Vous pouvez utiliser l'authentification SSO **unique** (*Single Sign-On*) pour vous connecter à votre compte OVHcloud. Pour activer ces connexions, votre compte et vos services ADFS (Active Directory Federation Services) doivent être configurés à l'aide des authentifications SAML (*Security Assertion Markup Language*).
+
+**Ce guide vous explique comment associer votre compte OVHcloud à un Active Directory externe.**
+
+## Prérequis
+
+- Les services ADFS (Active Directory Federation Services) doivent s'exécutent sur votre serveur
```
Comment (sebferrer) [inline pages/account/customer/ovhcloud-account-connect-saml-sso/guide.fr-fr.md]:
```suggestion
- Les services ADFS (Active Directory Federation Services) doivent s'exécuter sur votre serveur
```

File: pages/account/customer/ovhcloud-account-connect-saml-sso/guide.fr-fr.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+
+![Profil menu OVHcloud](images/ovhcloud_profile_menu.png){.thumbnail}
+
+Cliquez sur le bouton `Login SSO`{.action} .
+
+![OVHcloud connect SSO step 1](images/ovhcloud_user_management_connect_sso_1.png){.thumbnail}
+
+Renseignez les métadonnées XML de votre service ADFS. Le champ `Nom d'attribut de groupe` est facultatif dans ce cas. Cliquez sur `Confirmer`{.action}.
+
+![OVHcloud connect SSO step 2](images/ovhcloud_user_management_connect_sso_2.png){.thumbnail}
```
Comment (sebferrer) [inline pages/account/customer/ovhcloud-account-connect-saml-sso/guide.fr-fr.md]: Here you didn't translate, but it's maybe on purpose

**Comment by ovh-cds** [comment]
Failed to build static files:
```sh
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/translator.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.translations = yaml.load(fd)
/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_filters/banner.py:15: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  self.banners = yaml.load(fd)
WARNING: Empty alt attribute for image img_2903.jpg in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/web/domains/glue_registry/guide.nl-nl.md
WARNING: Empty alt attribute for image img_2900.jpg in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/web/domains/glue_registry/guide.nl-nl.md
WARNING: Empty alt attribute for image img_2901.jpg in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/web/domains/glue_registry/guide.nl-nl.md
WARNING: Empty alt attribute for image img_2902.jpg in /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/web/domains/glue_registry/guide.nl-nl.md
WARNING: Other images have empty alt attributes
  File "/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_markdown/blocksprocessor.py", line 303, in run
    tabs = self.build_tabs_object(lines)
  File "/tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/plugins/ovh_markdown/blocksprocessor.py", line 280, in build_tabs_object
    if line[0] != ">":
ERROR: Could not process /tmp/a6d15caf7e8c9d203af5aa134219bab2/run/docs-rendering/pages/account/customer/ovhcloud-account-connect-saml-sso/guide.es-es.md
  | ParserError: while parsing a block mapping
  in "<unicode string>", line 1, column 1:
    title: Activar las conexiones SS ... 
    ^
expected <block end>, but found '<scalar>'
  in "<unicode string>", line 3, column 86:
     ... e OVHcloud a través de SAML 2.0".
                                         ^
CRITICAL: RuntimeError: Error encountered
```

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @sebferrer

**Pull request merged**